test/harry/main/org/apache/cassandra/harry/model/ASTSingleTableModel.java

```diff
@@ -1539,9 +1539,10 @@ private List<PrimaryKey> findByPartitionEq(LookupContext ctx)
 
     private Clustering<ByteBuffer> key(Map<Symbol, Expression> values, ImmutableUniqueList<Symbol> columns)
     {
+        if (columns.isEmpty()) return Clustering.EMPTY;
```

Comment:
bug reported by @sunil9977 ; since i was already here quick to patch.  The bug is that not every table has clustering columns so we should return empty.

This wasn't noticed as CI doesn't run harry unit tests =(